builders/tooling/oracles/phala.md

```diff
@@ -0,0 +1,201 @@
+---
+
+# Launching Price Feeds with Phala
+
+## Introduction {: #introduction }
+
+[Phala Network](https://phala.network/){target=\_blank} is an off-chain compute network powered by [Secure Enclaves](https://docs.phala.network/developers/advanced-topics/blockchain-infrastructure#the-architecture){target=\_blank} that enables developers to build powerful smart contracts that connect to off-chain components called Phat Contracts. Phat Contracts are designed to enable functionality that surpasses the limitations of traditional smart contracts, such as storage, cost, and compute limitations while retaining trustlessness, verifiability, and permissionlessness. For more information about Phala's architecture, be sure to check out the [Phala docs](https://docs.phala.network/introduction/readme){target=\_blank}.
+
+Phala is not an oracle network itself; rather, Phala enables a variety of off-chain compute capabilities, such as a decentralized oracle network. Phala also provides a toolset called [Phala Bricks](https://bricks.phala.network/){target=\_blank} that makes it easy to quickly launch these types of features without having to build them from scratch. 
+
+This tutorial will walk through a demo of [interacting with price feeds](#fetch-price-data) enabled by Phat contracts on the demo EVM Appchain. Next, you'll learn how to [deploy price feeds to your own EVM Appchain](#launching-price-feeds-on-your-own-evm-appchain). Please be advised that the steps shown in this tutorial are for demonstration purposes only - it's highly recommended that you [contact the Phala team directly](https://dashboard.phala.network/){target=\_blank} as they can assist you with launching price feeds on your Appchain to ensure the integrity of the deployment process.
+ 
+## A Crash Course on Price Feeds {: #a-crash-course-on-price-feeds }
+
+Before interacting hands-on with price feeds, it's important to understand how price feeds work and how price feed data is accessed by smart contracts. In a standard configuration, each price feed is updated by a decentralized oracle network. Each oracle node is rewarded for publishing the price data to the [aggregator contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank}. 
+
+The aggregator contract receives periodic data updates from the network of oracles and aggregates and stores the data on-chain so that consumers can easily fetch it. However, the information is only updated if a minimum number of responses from oracle nodes are received (during an aggregation round). The end-user can retrieve price feeds with read-only operations via an aggregator interface or via a Consumer interface through the Proxy.
+
+![Price Feed Diagram](/images/builders/tooling/oracles/phala/phala-1.webp)
```

Comment:
The background for this image should be transparent so it looks good on dark mode too! I can help with this if you add the image to the drive 🙂

Reply:
Image was deleted from the page so we're good